mne/viz/_brain/_timeviewer.py

```diff
@@ -416,7 +425,6 @@ def play(self):
             self.time_call(idx, update_widget=True)
             if time_point == max_time:
                 self.playback = False
-        self.plotter.update()  # critical for smooth animation
```

Comment:
This always should have been indented one more level :( But if you now have `time_call` effectively trigger the `.update()`, then indeed it makes sense to remove it entirely now

Reply:
It was necessary after PyVista 0.24, probably something can be improved upstream